lib/bindings/HTTPBindings.js

```diff
@@ -135,7 +135,7 @@ function checkMandatoryParams(queryPayload) {
  * This middleware checks whether there is any polling command pending to be sent to the device. If there is some,
  * add the command information to the return payload. Otherwise it returns an empty payload.
  */
-function returnCommands(req, res, next) {
+function returnCommands(req, res) {
```

Comment:
Similarly to handleError() the solution should be to call `next();` in the proper place (I guess that just after L205, but not fully sure).

Reply:
fixed in a2e11f62c9516932f9f4e42afc7a52b915322831